src/react-webcam.js

```diff
@@ -298,7 +307,7 @@ export default class Webcam extends Component {
         muted={this.props.audio}
         className={this.props.className}
         playsInline
-        style={this.props.style}
+        style={this.props.mirror ? { ...this.props.style, transform: 'scaleX(-1)' } : this.props.style}
```

Comment:
The only problem that I can see with this currently, is that any existing `transform` property would be overwritten by the `scaleX(-1)`. Ideally, the `scaleX(-1)` would be appended if a transform property already existed.

Reply:
hmm in that case we could just replace the 
```js 
{ ...this.props.style, transform: 'scaleX(-1)' }
``` 
with 
```js
{ ...this.props.style, transform: this.props.style.transform ? `${this.props.style.transform} scaleX(-1)` }
```

Reply:
Will also probably have to check if the transform string already contains a scaleX(`replace here`)